src/common/helper.js

```diff
@@ -482,14 +482,16 @@ async function createResource (challengeId, memberHandle, roleId) {
  * @param {String} token The token
  * @returns
  */
-async function createSelfServiceProject (name, description, type, token) {
+async function createSelfServiceProject (name, description, type) {
   // const span = await logger.startSpan('helper.createSelfServiceProject')
 
   const projectObj = {
     name,
     description,
     type
   }
+
+  const token = await getM2MToken()
```

Comment:
This is not the right approach. Can you list which user roles/scopes that should be allowed to create a SS project - I'd recommend talking to the PO to get details on this.

Reply:
@rakibansary Any normal user should be able to create a SS work. Ref: https://topcoder.slack.com/archives/C03JNHSCG8N/p1666801053746829?thread_ts=1666765028.216809&channel=C03JNHSCG8N&message_ts=1666801053.746829

Reply:
Yeah, they pay online to create work and would never have any special permissions in the system.